tox.ini

```
@@ -71,7 +71,7 @@ commands =
 description = Run charm integration tests
 deps =
     pytest
-    juju
+    juju==2.9.38.1 # Latest juju 2
```

We can probably change it to:
```suggestion
    juju~=2.9.0
```
That way, we can get the library updates.